modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/PostProcessUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openapitools.codegen.utils;
+
+import org.openapitools.codegen.CodegenParameter;
+
+public class PostProcessUtils {
+  public static void postProcessParameter(CodegenParameter p) {
```

Comment:
What about putting this function in `AbstractJavaCodegen` class instead as this seems to be relevant to Java only?

Reply:
Or simply put the function in the `postProcessParameter` of `AbstractJavaCodegen` to avoid calling it in the Java client/server generator java classes ?